sdk/search/search-documents/test/public/utils/recordedClient.ts

```diff
@@ -11,28 +9,23 @@ import {
   SearchIndexClient,
   SearchIndexerClient,
 } from "../../../src";
-
-const isNode =
-  typeof process !== "undefined" &&
-  !!process.version &&
-  !!process.versions &&
-  !!process.versions.node;
-
-if (isNode) {
-  dotenv.config();
-}
+import { OpenAIClient } from "@azure/openai";
```

Comment:
I'm wondering the same thing. If it's not possible to deploy through an ARM template, it might be best to just generate the embeddings once and store them in the repo. What are your thoughts?